tsconfig.json

```diff
@@ -12,8 +12,8 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es6",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
-    "lib": ["ES6", "DOM"],                            /* Specify a set of bundled library declaration files that describe the target runtime environment. */
+    "target": "es2020",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
+    "lib": ["ES6", "DOM"],                               /* Specify a set of bundled library declaration files that describe the target runtime environment. */
```

Comment:
Should we add `ES2020` here too? Or even replace `ES6`?

https://www.typescriptlang.org/tsconfig/#high-level-libraries

Reply:
Fixed.